utils/get_latest_copr_build

```diff
@@ -1,28 +1,55 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
 
 import json
 import os
 import sys
 
 import copr.v3
 
+CONFIG_PATH = os.path.expandvars(os.getenv('_COPR_CONFIG', '~/.config/copr'))
+client = copr.v3.Client(copr.v3.config_from_file(path=CONFIG_PATH))
+
+# In case we are not able to retrieve from env vars, use defaults from the
+# configuration files.
 ownername = os.getenv('COPR_OWNER')
 projectname = os.getenv('COPR_PROJECT')
 packagename = os.getenv('COPR_PACKAGE')
-if not ownername:
-    ownername, projectname = os.getenv('COPR_REPO', '').split('/', 1)
+pkgrelease = os.getenv('PKG_RELEASE')
+if not ownername or not projectname:
+    copr_repo = os.getenv('COPR_REPO', '').split('/', 1)
+    if len(copr_repo) == 2:
+        ownername, projectname = os.getenv('COPR_REPO', '').split('/', 1)
+    else:
+        # We have just the name of project inside COPR_REPO value, it is
+        # expected when user is owner of the project. So set the user as
+        # owner from the config file.
+        projectname = copr_repo[0]
+        ownername = client.config['username']
 
-CONFIG_PATH = os.path.expandvars(os.getenv('_COPR_CONFIG', '~/.config/copr'))
-client = copr.v3.Client(copr.v3.config_from_file(path=CONFIG_PATH))
+# limit set to 10 as it is not expected that there would be more packages build
+# in such short time
 builds = client.build_proxy.get_list(
     status='succeeded',
-    pagination={'limit': 1, 'order': 'id', 'order_type': 'DESC'},
+    pagination={'limit': 10, 'order': 'id', 'order_type': 'DESC'},
     ownername=ownername,
     projectname=projectname,
     packagename=packagename)
 
-if builds:
+for build in builds:
+    # Version in COPR contains VERSION-RELEASE string. We need just the
+    # release. Additionally we do not care about *elX* suffig. So remove it
+    # when compare.
+    _release = build['source_package']['version'].split("-")[-1]
+    # FIXME: instead of rsplit(.el8) remove whole elx.* as it can contain
+    # later e.g. .el8_0 instead of .el8
```

Comment:
Not sure that this is a thing in COPR

Reply:
Not sure. But better to mention it. Maybe the comparison would be done just using
```
    if _release in pkgrelease
    # or
    if _release.startswith(pkgrelease)
```

Regarding the schema we are using, this would be enough as well.